src/lib.rs

```diff
@@ -79,7 +83,7 @@ pub trait UdpStack {
 	/// If a packet has not been received when called, then
 	/// [`nb::Error::WouldBlock`](https://docs.rs/nb/1.0.0/nb/enum.Error.html#variant.WouldBlock)
 	/// should be returned.
-	fn read(
+	fn receive(
```

Comment:
Nitpick: Berkley sockets specifies this as `recv()` (and later `send_to` as `sendto`).

I personally don't care and am okay with `receive()` and `send_to()`. Just wanted to point this out.

Reply:
I considered that, but I wanted to stick with the more common Rust name style